### ISSUE TITLE
TypeError: record.start is not a function 

### ISSUE THREAD

**Issue by rithiksachdev** [gillesdemey/node-record-lpcm16#59]
Could not find a declaration file for module 'node-record-lpcm16'. '/home/rithik/Desktop/speechtotext/node_modules/node-record-lpcm16/index.js' implicitly has an 'any' type.
  Try `npm install @types/node-record-lpcm16` if it exists or add a new declaration (.d.ts) file containing `declare module 'node-record-lpcm16';`ts(7016)

**Comment by mccarthydanielle** [comment]
> Could not find a declaration file for module 'node-record-lpcm16'. '/home/rithik/Desktop/speechtotext/node_modules/node-record-lpcm16/index.js' implicitly has an 'any' type.
> Try `npm install @types/node-record-lpcm16` if it exists or add a new declaration (.d.ts) file containing `declare module 'node-record-lpcm16';`ts(7016)

@rithiksachdev I am encountering the same issue. Were you able to resolve it?

**Comment by rithiksachdev** [comment]
Yes, @mccarthydanielle. I actually don't remember the complete context of what I was trying to do and how I came across this error. can you explain what you are trying to do and what command led to this error? please mention your system specifications so that I can help you. (OS, OS version, Node version, npm version )

**Comment by mccarthydanielle** [comment]
@rithiksachdev Thanks for your response! I ended up not using this library 🙂
